apps/api/src/modules/users/users.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { Controller, Post, Body, Logger, HttpException } from '@nestjs/common';
+import { JsendFormatter } from 'src/common/jsend-formatter';
+import { UsersService } from './users.service';
+import { CreateUserInput } from './dto/create-user.input';
+
+@Controller('users')
+export class UsersController {
+  constructor(
+    private readonly usersService: UsersService,
+    private readonly jsend: JsendFormatter,
+    private logger: Logger,
+  ) {}
+
+  @Post()
+  async addUser(@Body() userData: CreateUserInput): Promise<Record<string, unknown>> {
```

Comment:
We don't yet want an endpoint for creating new users. This can be done later and will also need to be protected by a Guard to only allow Admin users to create new users.

Reply:
Ok, will remove